op.yml

```diff
@@ -50,7 +50,7 @@ outputs:
   id:
     number:
       description: ID of the created release, can be used as input for other github.release ops
-version: 1.2.0
+version: 2.0.0
```

Comment:
When you `opctl run build` it will re-generate the README.md. Basically start [at step 2](https://github.com/opspec-pkgs/project/blob/main/how-tos.md#step-2) for edits. Highly recommend to update [this](https://github.com/opspec-pkgs/github.release.create/blob/eb781b38af8cdab81205ec6c2096087a0d031824/.opspec/build/op.yml#L17) to latest too; bootstrap updates all the ops and this one looks out of date. 

Aside) Usually we'll run [_.op.repos.standardize](https://github.com/opspec-pkgs/_.op-repos.standardize/blob/main/op.yml) when the bootstrap op changes to apply it to all repos but it hasn't been done recently so that's why that reference is out of date. That op automates alot; clones all op repos, runs bootstrap on them, commits and pushes to Github, but you still need to submit a PR and approve and release. We probably could optimize that workflow more.

Reply:
I've pushed the results of the `_op.repos.standardize` run to this PR and I've also run `opctl run build` and committed those results. I think that implements all of the feedback recommended in your comment?

Reply:
nice! One new comment for your consideration on the github action